deploy/helm/sumologic/templates/setup/setup-configmap.yaml

```diff
@@ -128,6 +134,13 @@ data:
         collector_id = "${sumologic_collector.collector.id}"
     }
 
+    {{- if .Values.sumologic.traces.enabled }}
+    resource "sumologic_http_source" "traces_source" {
```

Comment:
I think you need to set Content-type here for making tracing type source otherwise this will make normal HTTP source cc @tanayjha

Reply:
@Gourav2906 We are going to use `application/vnd.sumologic.zipkin`

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/conf/traces/traces.conf#L16

Do we need to add sth to terraform too?

Reply:
Yes we need to add `contentType` and we don't need to add `application/vnd.sumologic.zipkin` header anymore. 
So the json will look like to create a Trace source:
```
{
    "api.version": "v1",
    "source": {
        "messagePerRequest": false,
        "sourceType": "HTTP",
        "contentType": "Zipkin",
        "name": "TraceTrySource"
    }
}
```
(Notice the contentType field set to **Zipkin**)
This will return a trace endpoint, which will look like: `http://localhost:23660/receiver/v1/trace/Za***`
You can directly send data to this endpoint and it would accepted as trace (no need to add any headers)